app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
+  belongs_to :post, foreign_key: :post_id
+
```

Comment:
- [Optional] Consider adding an after_save method to update likes counter after the like has been saved. Also consider making the update counter method private since you don't need to access it outside this class.

Reply:
okay. I will make changes right away. Thanks for your time and effort